src/React/Interpret.hs

```diff
@@ -86,8 +85,11 @@ interpret' cb = \case
     Parent f as hs children -> do
         children' <- forM children (interpret' cb)
         let hs' = map (unHandler cb) hs
-        element f as hs' children'
+        node <- element f as hs' children'
```

Comment:
That line change and the case change shouldn't really be there - I think I had some `putStrLn`s there at an earlier stage while trying to figure out #16 . Happy to change them back.